apps/api/docs/database-design.md

```diff
@@ -0,0 +1,49 @@
+# **Database Design**
+
+This document provides a comprehensive and structured overview of the database design for Transcript Summarization App.
+
+It serves as a reference guide for stakeholders, including developers, database administrators, project managers, and other parties involved in the development, maintenance, and understanding of the application's database.
+
+The tables discussed below will be created as part of the database migration.
+
+## **Database Schema**
+
+The database schema consists of the following 3 tables:
+
+- **User_files :** Contains details about all the user uploaded files with their Task IDs.
+- **Job_logs** : Contains details about the queue system-related data, such as job metadata and status.
+
+This schema can be visualized in the following image:
+![Block Diagram](./assets/transcript-summarization-database-schema.jpg)
+
+## **Data Dictionary**
+
+### User Files
+
+| Attribute | Data Type | Not Null | Default | Description |
+| --- | --- | --- | --- | --- |
+| task_id | int(11) | True |     | Primary key, unique identifier for the uploaded record |
+| input_file_path | varchar(255) |     |     | File path of user uploaded file |
+| output_file_path | varchar(255) |     |     | File path of summarized file stored in server after successful processing |
```

Comment:
This is incorrect.
files table will just have file details, it doesn't has any concept of input and output files.
1 record corresponds to a particular file.

Reply:
Removed the input and output file path fields from files table.